src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -72,6 +77,12 @@ public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public void validateAnswerOwner(NsUser loginUser) throws CannotDeleteException {
+        if (!this.isOwner(loginUser)) {
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+        }
+    }
+
```

Comment:
현재는 valid와 updateDeleted가 분리되어 구현되어 있음.
이 둘을 하나로 구현하는 아래와 비교했을 때 어느 방식으로 구현하는 것이 더 좋을까?

```suggestion
public void delete(NsUser loginUser) throws CannotDeleteException {
    // 유효성 체크와 삭제 상태로 변경
}
```

Reply:
이 피드백 반영했는지 확인 필요